spot_examples/spot_examples/simple_sub.py

```diff
@@ -0,0 +1,55 @@
+# This file interfaces with spot to send action. Helper files are robot_commander and simple_spot_commander
+import os
+from typing import List
+from contextlib import closing
+from std_msgs.msg import String
+from geometry_msgs.msg import Pose, PoseArray
+
+from synchros2.subscription import Subscription
+import synchros2.process as ros_process
+from .robot_commander import RobotCommander
+from ament_index_python.packages import get_package_share_directory
+
+
+@ros_process.main()
+def main() -> None:
+    print("here")
+    robot_name= os.getenv("ROBOT_NAME")  # Get robot name from environment variable
+    robotcommander = RobotCommander(robot_name=robot_name)
```

Comment:
nit: to keep the casing more consistent
```suggestion
    robot_commander = RobotCommander(robot_name=robot_name)
```